projects/plugins/jetpack/modules/subscriptions.php

```diff
@@ -1034,3 +1035,49 @@ public function register_post_meta() {
 Jetpack_Subscriptions::init();
 
 require __DIR__ . '/subscriptions/views.php';
+
+/**
+ * Gate access to posts
+ *
+ * @param string $the_content Post content.
+ *
+ * @return string
+ */
+function check_for_paid_subscription( $the_content ) {
```

Comment:
I think this function should be something like `locked_content`

Reply:
Fixed in e5d97e3815a83923b7bbe252b5c86d49a9a5297b. Change to `maybe_get_locked_content`